.vscode/settings.json

```diff
@@ -16,5 +16,9 @@
     },
     "yaml.schemas": {
         "https://json.schemastore.org/golangci-lint.json": "file:///home/nandor/projects/dyrectorio/dyo-pub/.golangci.yml"
```

Comment:
We need your home path in here @nandor-magyar ?

Reply:
Sorry.